packages/components/src/InlineEdit/InlineEdit.mdx

```diff
@@ -0,0 +1,171 @@
+---
+name: InlineEdit
+menu: Components
+route: /components/inline-edit
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { InlineEdit } from ".";
+
+# Inline Edit
+
+```ts
+import { InlineEdit } from "@jobber/components/InlineEdit";
+```
+
+<Playground>
+  <InlineEdit text="Bob" />
+</Playground>
+
+<!--
+  INTERFACE: This is for the proposal stage of the component
+
+  Provide an example of what the component looks like in code. How would you use
+  it from another React component? This should consist primarily of code blocks.
+  This section can be deleted once the component is built, when the Playground
+  will demonstrate this at a higher fidelity.
+-->
+
+[coming soon]
+
+## Props
+
+<Props of={InlineEdit} />
+
+<!--
+  It is not necessary to provide an example of each prop. Use usage examples to
+  highlight key features of a component or particular considerations.
+
+  See `Button.mdx` for a good example of this.
+-->
+
+[coming soon]
+
+## Usage Guidelines
+
+<!--
+  What is the design purpose of this component? How do its responsibilities
+  relate to other components? What should this component not be used for? What
+  are some related components in Atlantis? Try to describe this component not
+  in terms of what it looks like or what elements make it up, but what function
+  it performs or how it enables a user to perform tasks (e.g. “Buttons allow users
+  to initiate, complete, and reverse actions.” vs “Buttons are rectangular elements
+  that have a green background and white text”).
+-->
+
+The `<InlineEdit>` will be used to switch between displaying a value and displaying the component to update or change the value. The user can interact with a single data point in a **read** state that transforms into an input field for **edit** state, with an associated **save** and **cancel** action to update that one piece of data. 
+
+InlineEdit will be used for modifying a single piece of data/information that could be presented as content on its own or may have a default setting. It will have two states a **read** and an **edit** state. The **read** state will show as clearly editable data field with an 'edit’ icon next to data. The **show** state will have a small input and a way to ‘cancel’ any changes made to the data.
+
+Scenarios where InlineEdit should not be used include the following:
+- Should not be used inside of a modal 
+- Should not be used to replace **forms** as a primary means of data entry (e.g. first time create) 
+- Should not be used with an Input Group 
+- Should not be used in cases where there are multiple settings changes (2+ fields)
+
+## Other Considerations
```

Comment:
Two notes here:

- I think this fits well as a "related components" section
- I think this fits _under_ the design guidelines, so let's bump the headings in this section down a level

Reply:
(Otherwise this could go into the Notes section since some of it talks about planned future functionality)

Reply:
@chris-at-jobber so move lines 64 & 65 to a different section?

Reply:
Summing up from our chat:

- let's change this title of "Other considerations" to "Related Components"
- Change this from an h2 to an h3
- Change the two child headers from h3 to h4